### ISSUE TITLE
Does this fork support mobile device?

### ISSUE THREAD

**Issue by xeoshow** [6pac/SlickGrid#74]
Hi,

I am new to this fork of SlickGrid, does it support touch event so could work on mobile device?

Thanks a lot!

**Comment by 6pac** [comment]
Hi @xeoshow,
this is essentially an active and up to date version of MLeibman's repo.
i don't think mobile devices have been given high priority. But if you are keen to get the grid working in a mobile environment, I'm happy to offer help with problems that you come across.

**Comment by xeoshow** [comment]
Hi,
Thanks for reply, I am just trying to choose a grid component for developing my product, and found this is an active fork of slickgrid. And I found below fork of slickgrid, not sure if it is merged in 6pac branch.
https://github.com/reebalazs/SlickGrid-touch

Thanks a lot!

**Comment by 6pac** [comment]
Thanks for the link, I hadn't seen that version! I will have a look. It was probably forked a while back, and that could make it hard to merge, though...

**Comment by xeoshow** [comment]
Below are the more background information related:
http://www.slideshare.net/reebalazs/slickgrid-touch-making-complex-javascript-widgets-work-on-mobile-devices
http://stackoverflow.com/questions/11500842/slickgrid-mobile-look
http://stackoverflow.com/questions/10051694/slickgrid-in-the-android-browser-inner-div-scrolling

Slickgrid has a solid codebase, hope it could also be used in the mobile devices...

**Comment by 6pac** [comment]
Just FYI, the reebalazs repo changes are all to do with Bootstrap compatibility, and nothing to do with touch (although Bootstrap does bring a certain amount of touch compatibility, but it's a pretty major change as jQueryUI is a fundamental part of the grid and there are well document conflicts between jQueryUI and Bootstrap). 

The links in Slideshare are all dead, as is the repo for FastBreak.

The two StackOverflow questions are quite old, and I was able to fire up the grid sample pages on my reasonably old Android phone without serious problems. The main issue is that of 'fat finger' given the amount of detail displayed on a small screen. Perhaps we have moved past these issues.

So I'm not sure if this approach is actually needed or has been successful. It is debatable whether something as complex as SlickGrid should even be attempted on a mobile device. If it was, the generalised display and touch issues are probably the most pressing, and these are not specific to SlickGrid. I think a lot of thought and experimentation are needed.
